### PR TITLE
Fix TypeError in CPU class for Linux

### PR DESCRIPTION
**Pull request by koraktor**
CPU was transformed into a class in 8d995e961f549e555f405d2567235dab53f6baad.

### CODE REVIEW

File: Library/Homebrew/extend/os/linux/hardware/cpu.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module Hardware
   class CPU
-    extend self
+    class << self
```
Comment (mikemcquaid) [inline Library/Homebrew/extend/os/linux/hardware/cpu.rb]: Everything below this needs indented and an extra `end` may be required.

Comment (koraktor) [inline Library/Homebrew/extend/os/linux/hardware/cpu.rb]: Sorry, you're right. Did only a quick fix on GitHub.

Comment (mikemcquaid) [inline Library/Homebrew/extend/os/linux/hardware/cpu.rb]: No worries, breakage is my bad so appreciate the fix.

Comment (koraktor) [inline Library/Homebrew/extend/os/linux/hardware/cpu.rb]: Fixed.

**Comment by mikemcquaid** [comment]
Thanks @koraktor!

**Pull request merged**